### PR TITLE
Model regression tests - show last performance on master

### PR DESCRIPTION
**Pull request by tczekajlo**
Solves #6393

### CODE REVIEW

**Comment by tczekajlo** [comment]
/modeltest

```yml
include:
 - dataset: ["Carbon Bot"]
   config: ["Sparse + DIET(bow) + ResponseSelector(bow)"]

```

**Comment by tczekajlo** [comment]
@dakshvar22 Can you 👀  if you have a minute?

**Comment by dakshvar22** [comment]
@tczekajlo The format looks good 👍 I'll probably add a small dummy change today to one of our components to shoot down the accuracy and test if the difference comes up correctly.

**Comment by dakshvar22** [comment]
@tczekajlo Doesn't look like that worked 🤔

File: rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py [hunk 1]
```diff
@@ -127,8 +127,8 @@ def bert_embeddings_post_processor(
     Returns:
         sentence level embedding and post-processed sequence level embedding.
     """
-    sentence_embedding = sequence_embeddings[0]
-    post_processed_embedding = sequence_embeddings[1:-1]
+    sentence_embedding = np.zeros_like(sequence_embeddings[0])
+    post_processed_embedding = np.ones_like(sequence_embeddings[1:-1])
```
Comment (dakshvar22) [inline rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py:131]: We should revert these back to original before merging the PR

**Comment by tczekajlo** [comment]
> @tczekajlo Doesn't look like that worked 🤔

Hmm, let me fix it 😄

**Comment by tczekajlo** [comment]
@dakshvar22 Now, it looks better, can you 👀  if you have a minute?

**Comment by dakshvar22** [comment]
The format and output looks good to me! :rocket: 
Haven't reviewed the actual implementation as I am not familiar with the code.

**Comment by tczekajlo** [comment]
> The format and output looks good to me! 🚀
> Haven't reviewed the actual implementation as I am not familiar with the code.

Cool, I wanted to be sure that format and output are ok. I'll catch someone else to take a look at the implementation part 😄

File: .github/workflows/ci-model-regression.yml [hunk 2]
```diff
@@ -460,6 +460,22 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Download an artifact with the results for the schedule version
+        run: |
+          # Schedule workflows ID - 1498790
```
Comment (ricwo) [inline .github/workflows/ci-model-regression.yml:465]: is there a way we can fetch the ID for the _latest_ or _current_ version of `ci-model-regression-on-schedule.yml`? this is not ideal as it needs to be manually updated if the other file changes.

if it has to happen via ID, we need to add comments here that the ID has to be updated if `ci-model-regression-on-schedule.yml` changes, and also in `ci-model-regression-on-schedule.yml` to update the ID in this file

Comment (ricwo) [reply .github/workflows/ci-model-regression.yml:465]: also we need to think about the case where another PR changes `ci-model-regression-on-schedule.yml`, so simply getting the latest ID wouldn't be right. can we fetch the latest workflow ID of that workflow for the current branch?

**Review by ricwo — COMMENTED** [review]
Implementation looks really good 👍  it would be great if we could do something about that static workflow ID

File: .github/workflows/ci-model-regression.yml (hunk 2 above)
Comment (tczekajlo) [reply .github/workflows/ci-model-regression.yml]: The IDs for workflows are pretty static, they don't change even there are several branches that have different versions of the same workflow, additionally, schedule workflows can exist only in the master branch.

I've changed a bit this part and now, we fetch ID via GH API for the scheduled workflow.

**Review by ricwo — APPROVED** [review]
Awesome, thanks for making those changes 🚀

**Pull request merged**